napari/_qt/_qapp_model/_tests/utils.py

```diff
@@ -0,0 +1,25 @@
+import qtpy
+from qtpy.QtWidgets import QMenu
+
+
+def get_submenu_action(window_menu, submenu_text, action_text):
+    def _get_menu(act):
+        # this function may be removed when PyQt6 will release next version
+        # (after 6.3.1 - if we do not want to support this test on older PyQt6)
+        # https://www.riverbankcomputing.com/pipermail/pyqt/2022-July/044817.html
+        # because both PyQt6 and PySide6 will have working manu method of action
+        return (
+            QMenu.menuInAction(act)
+            if getattr(qtpy, 'PYQT6', False)
+            else act.menu()
+        )
+
+    actions = window_menu.actions()
+    for action1 in actions:
+        if action1.text() == submenu_text:
```

Comment:
I was recently looking at this function and I think a `findAction` would work here?

Reply:
Actually, I can see that the current action parameters (submenu_text and action_text) have a nice consistency so we can keep this as is.

What would be nice is if `findAction` worked for finding submenu actions so we didn't have to do this.